roles/pulp-webserver/templates/nginx.conf.j2

```diff
@@ -81,9 +81,5 @@ http {
 
         include conf.d/*;
 
-        location / {
```

Comment:
I had to remove it because I was getting error when I was trying to define root location on galaxy_ng

Reply:
Do we need to do something similar from the apache config?

Reply:
Actually I've been trying to make apache work in many ways, but so far I was not able to do that